core/tests/protractor_utils/PreferencesPage.js

```diff
@@ -34,6 +34,13 @@ var PreferencesPage = function() {
   var subscriptions = element.all(by.css('.protractor-test-subscription-name'));
   var systemLanguageSelector = element.all(
     by.css('.protractor-test-system-language-selector')).first();
+  var bioLink = element(by.css('.protractor-test-preferences-editBio'));
+
+  this.addBio = function(bio) {
+    bioLink.click();
+    bioLink.sendKeys(bio);
+    pageHeader.click();
```

Comment:
Why do we have to click pageHeader? (You can check ExplorationEditorSettingsTab.setTitle e2e test for similar behavior)

Reply:
I have written about this problem in Oppia-Chat:
"When I add a bio manually, it autosaves (since there is no button to Save or Update Info). However, when the script performs the same action (adding bio), and refreshes the page, the bio is not saved. Any suggestions how to solve this problem?"
When script clicks the pageHeader(any other area of webpage), then bio is saved.
How can I solve this problem other way?

Reply:
@yesass, Have you gone through the  `ExplorationEditorSettingsTab.setTitle`? We have similar behavior in the exploration settings tab, maybe reading those code will help you understand better. Let me know if it's not clear. Thanks!

Reply:
@DubeySandeep Hi! I have explored it. However, clearing the text area and then sending keys does not help. Did I understood your hint clearly?